cogs/fun.py

```
@@ -0,0 +1,299 @@
+import discord
+from discord.ext import commands
+import random
+import aiohttp
```

Ruff (0.11.9)

`aiohttp` imported but unused

Remove unused import: `aiohttp`

(F401)